src/__tests__/CardDetails.test.tsx

```diff
@@ -0,0 +1,23 @@
+import '@testing-library/jest-dom';
+
+import { CardDetails } from '@components/';
+import { store } from '@store/store';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
```

Comment:
Я бы разнес импорты из библиотек и импорты из твоих файлов - иначе каша. А в компонентах еще есть стили, типы, картинки...

```suggestion
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import { render, screen } from '@testing-library/react';
import { beforeEach, describe, expect, it, vi } from 'vitest';

import { store } from '@store/store';
import { CardDetails } from '@components/';
```